pio/src/iSpindel.cpp

```diff
@@ -788,7 +788,7 @@ bool uploadData(uint8_t service)
     sender.add("brand", "wemos_d1_mini");
     sender.add("version", FIRMWAREVERSION);
     sender.add("chipid", chipidHashed);
-    sender.add("s_number_wort_0", (float)(round(Gravity * 10) / 10));
+    sender.add("s_number_wort_0", (float)(round(Gravity * 10000) / 10000));
```

Comment:
Can we consider to send a raw values (without truncation) instead of such fixes with changing a magic numbers?

Reply:
done! 👍